sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -2,19 +2,20 @@
 import logging
 import os
 import re
+import sys
 from datetime import datetime
 from pathlib import Path
 
 import boto3
 from sqlalchemy.orm import Session
 
-from .SDSCode.database import database as db
-from .SDSCode.database import models
-
-# Setup the logging
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger(__name__)
+from .database import database as db
+from .database import models
 
+# Logger setup
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
Why are these set separately? I think we can get rid of the basicConfig and sys import and put this logger to debug.
```suggestion
logger.setLevel(logging.DEBUG)
```

Reply:
That's how we had in indexer and it's been able to log. I recall batch job logging troublesome. I am testing it on AWS.

Reply:
Ok. That seems to work. It just produced long logs that I haven't seen before. I see INFO logs I usually see. 
<img width="1440" alt="Screen Shot 2024-01-29 at 9 29 16 AM" src="https://github.com/IMAP-Science-Operations-Center/sds-data-manager/assets/36522642/60c28970-5431-4283-8b32-0b3d70ac7b0e">